src/components/DrawerPage/DrawerPage.tsx

```diff
@@ -0,0 +1,63 @@
+import React, { useState } from 'react';
```

Comment:
add ElementType to the import, please :) ...

Reply:
As below.